distributed/worker.py

```diff
@@ -1842,6 +1846,7 @@ def gather_dep(self, worker, dep, deps, total_nbytes, cause=None):
 
                 total_bytes = sum(self.nbytes.get(dep, 0) for dep in response["data"])
                 duration = (stop - start) or 0.5
+                bandwidth = total_bytes / (duration - abs(self.scheduler_delay))
```

Comment:
There is a non-zero chance this results in a division by zero.

I am wondering if we could simply leave out the `abs(self.scheduler_delay))` part?

* the bandwidth is an approximation anyway
* the code would be more robust
* it would be consistent with the metrics computed below

Reply:
Thanks for the comment @StephanErb .  What you say makes sense to me.  I've removed the scheduler_delay change.